src/Core/src/Platform/Windows/SearchBarExtensions.cs

```diff
@@ -16,110 +35,85 @@ public static void UpdatePlaceholder(this AutoSuggestBox platformControl, ISearc
+				{
+					resource.Remove(key);
+				}
+			}
+			else
+			{
+				foreach (var key in keys)
+				{
+					resource[key] = brush;
+				}
+			}
 		}
 
 		public static void UpdateFont(this AutoSuggestBox platformControl, ISearchBar searchBar, IFontManager fontManager) =>
 			platformControl.UpdateFont(searchBar.Font, fontManager);
 
-		public static void UpdateHorizontalTextAlignment(this AutoSuggestBox platformControl, ISearchBar searchBar, MauiSearchTextBox? queryTextBox)
+		public static void UpdateHorizontalTextAlignment(this AutoSuggestBox platformControl, ISearchBar searchBar)
 		{
-			if (queryTextBox == null)
-				queryTextBox = platformControl.GetFirstDescendant<MauiSearchTextBox>();
-
-			if (queryTextBox == null)
-				return;
-
-			queryTextBox.TextAlignment = searchBar.HorizontalTextAlignment.ToPlatform();
+			platformControl.HorizontalContentAlignment = searchBar.HorizontalTextAlignment.ToPlatformHorizontalAlignment();
 		}
 
-		public static void UpdateVerticalTextAlignment(this AutoSuggestBox platformControl, ISearchBar searchBar, MauiSearchTextBox? queryTextBox)
+		public static void UpdateVerticalTextAlignment(this AutoSuggestBox platformControl, ISearchBar searchBar)
 		{
-			if (queryTextBox == null)
-				return;
-
-			queryTextBox.VerticalAlignment = searchBar.VerticalTextAlignment.ToPlatformVerticalAlignment();
+			platformControl.VerticalContentAlignment = searchBar.VerticalTextAlignment.ToPlatformVerticalAlignment();
 		}
 
-		public static void UpdateMaxLength(this AutoSuggestBox platformControl, ISearchBar searchBar, MauiSearchTextBox? queryTextBox)
+		public static void UpdateMaxLength(this AutoSuggestBox platformControl, ISearchBar searchBar)
 		{
-			if (queryTextBox == null)
-				queryTextBox = platformControl.GetFirstDescendant<MauiSearchTextBox>();
-
-			if (queryTextBox == null)
-				return;
-
-			queryTextBox.MaxLength = searchBar.MaxLength;
-
 			var currentControlText = platformControl.Text;
 
 			if (currentControlText.Length > searchBar.MaxLength)
 				platformControl.Text = currentControlText.Substring(0, searchBar.MaxLength);
 		}
 		
-		public static void UpdateIsReadOnly(this AutoSuggestBox platformControl, ISearchBar searchBar, MauiSearchTextBox? queryTextBox)
-		{
-			if (queryTextBox == null)
-				queryTextBox = platformControl.GetFirstDescendant<MauiSearchTextBox>();
-
-			if (queryTextBox == null)
-				return;
-
-			queryTextBox.IsReadOnly = searchBar.IsReadOnly;
-		}
-
-		public static void UpdateIsTextPredictionEnabled(this AutoSuggestBox platformControl, ISearchBar searchBar, MauiSearchTextBox? queryTextBox)
+		public static void UpdateIsReadOnly(this AutoSuggestBox platformControl, ISearchBar searchBar)
 		{
-			if (queryTextBox == null)
-				queryTextBox = platformControl.GetFirstDescendant<MauiSearchTextBox>();
-
-			if (queryTextBox == null)
-				return;
-
-			queryTextBox.IsTextPredictionEnabled = searchBar.IsTextPredictionEnabled;
+			platformControl.IsEnabled = searchBar.IsReadOnly;
```

Comment:
`ReadOnly` also not supported/exist on `AutoSuggestBox`, assumed `IsEnabled` could match, but SearchBar has `IsEnabled` property which is also mapped, maybe remove this mapping CC @mattleibow

Reply:
I think this is a good approach for now.